client/app/styles/onboarding.styl

```diff
@@ -58,7 +75,51 @@ footer
 
 
 // FORMS
+
+form
+    font-size: .9em
+
+    .line
+        position: relative
+
+        button[action="password-visibility"]
+            position:   absolute
+            top:        0
+            right:      0
+
+        button.icon
+            background-size: 1em 1em
+            margin: .55em 0
+
+            span
+                display: none
+
+        .hide-password
+            background: aqua url(/app/assets/sprites/hide-eye-icon.svg)
```

Comment:
It can't work here, as CSS doesn't do any require from assets. If we want to do sprinting, we must include the files in the template using a `svg: use(xlink:href)` syntax (which is perfectly valid for an iconic button).